cloudflare.loader.php

```diff
@@ -83,6 +83,7 @@
     'deleted_post',                     // Delete a post
     'edit_post',                        // Edit a post - includes leaving comments
     'delete_attachment',                // Delete an attachment - includes re-uploading
+    'post_update',                      // Update a post
```

Comment:
Could you link which hook this is?

Reply:
Sorry…just pushed a change to fix the typo. Here’s the hook: https://developer.wordpress.org/reference/hooks/post_updated/